### ISSUE TITLE
Redundant properties for each uploaded product (if assigned to a category)

### ISSUE THREAD

**Issue by marcofranke** [nimble-platform/common#12]
Each specific product (in Marmotta) is an Item and has a reference to a concept or eclass.  Beyond this reference, there are properties like name or price which are defined both in Item and within the referenced concept. To have just one occurrence of the "same" property, we need a rule which property has to be used and which have to be ignored.

**Comment by iot365** [comment]
In the "normal" case, the user should just add properties once for each product. If the same property is specified more than once using different concepts/names, we should already catch this by input validation, if possible.

**Comment by marcofranke** [comment]
Does it means there will be a check in the index: properties which checks whether a property has a value for any induvidual?  If yes,  I get the property and if not it isn't included in the index?  then,  we could seperate redundant properties like name or price in meaningful and meamingless for the product search.

**Comment by iot365** [comment]
Yes, within the catalogue2 index, only properties that have been used in any catalogue will be available - otherwise the currently implemented LDPath statement does not find a match. I think that it makes sense for the user, if only "existing" properties can be selected (as in the classical faceted search).
